### DISCUSSION TITLE
Autopairs prevent movement with curly braces?

### DISCUSSION THREAD

**Discussion by adamsitar** [LazyVim/LazyVim#4546] [General]
I've noticed that when I try to go up by a blank line with `{`, the autopair plugin instantly inserts `}` and moves me down where I was, so to enable this movement, autopairs have to be disabled 

Is this an issue anyone else is having? Seems like bad default configuration of autopairs in lazyvim...

**Comment by dpetka2001** [comment]
Looks like something wrong with your configuration because LazyVim default installation doesn't have this behavior. 

Following screencast with default LazyVim installation

[Screencast_2024-10-05-23-27-14.webm](https://github.com/user-attachments/assets/61f96f23-2dfd-4259-8cb1-16c662429384)
